### PR TITLE
Add Blog

### PR DESCRIPTION
**Pull request by PXingwei**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by PXingwei** [comment]
Thanks @lijiarui for reviewing my blog! I used the wrong username to commit the blog and have problems in signing CLA. I had to closed the previous pull request. I also created a new folder for the image. Thanks!

File: _posts/2020-06-04-group-wechat-bot.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "微信群助手"
```
Comment (lijiarui) [inline _posts/2020-06-04-group-wechat-bot.md]: Please change a more clear title for this blog, then it may attract more readers.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-06-04-group-wechat-bot.md (hunk 1 above)
Comment (PXingwei) [reply _posts/2020-06-04-group-wechat-bot.md]: Thanks for pointing out!

**Comment by PXingwei** [comment]
Hi @lijiarui, I have changed the title, are there other changes that I need to make?

**Review by lijiarui — APPROVED** [review]
Good job!

**Comment by lijiarui** [comment]
Please resolve the conflict and then I can merge this PR.

**Comment by lijiarui** [comment]
duplicate with #278

**Pull request closed without merging**